lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
Small formatting thing I just noticed: can we remove the spaces between heads of the same function? The general rule for Erlang and Elixir is that there's only a newline between completely separate functions.